api/courses/views.py

```diff
@@ -126,6 +126,14 @@ def semester_dept(request, semester_code, dept_code):
     return JSON(dept.toJSON())
 
 
+def semester_reviews(request, semester_code):
```

Comment:
Does this endpoint provide information that we wouldn't want to have a normal token user easily access/do you need to add this code?
```python
    if not request.consumer.access_pcr:
        # This method is only available to those with review data access.
        raise API404("This is not the database dump you are looking for.")
```

I think we limit some expensive endpoints to only a select group of users so that it doesn't slow down PCR. Otherwise, LGTM!